packages/newspringchurchapp/src/prayer/data/queries/getCampusPrayerRequests.js

```diff
@@ -18,6 +18,10 @@ export default gql`
         id
         name
       }
+      prayerHelpContent {
```

Comment:
I think I'd like to see this queried above these components (in their parent) and passed down to each list of prayers. That is an easy win for less queries.

Reply:
...yea. This method is what was suggested but I think I agree with you. I had the same thought with not as much conviction.